model/cloud.go

```diff
@@ -13,6 +13,8 @@ const (
 	InviteLimitation               = "invite"
 )
 
+var MockCWS string
```

Comment:
Did you check using boolean instead of string?

Reply:
Hello @ethervoid , ```-X``` works only for strings

Reply:
Right!